core/src/main/java/io/questdb/griffin/SqlKeywords.java

```diff
@@ -642,4 +642,37 @@ public static boolean isColumnKeyword(CharSequence tok) {
                 && (tok.charAt(i++) | 32) == 'm'
                 && (tok.charAt(i) | 32) == 'n';
     }
+
+    public static boolean isMaxKeyword(CharSequence tok) {
```

Comment:
we probably do not need these methods anymore?

Reply:
done